airbyte-ci/connectors/pipelines/tests/test_builds/test_python_connectors.py

```diff
@@ -31,9 +33,17 @@ def test_context_with_connector_without_base_image(self, test_context):
     def connector_with_base_image(self, all_connectors):
         for connector in all_connectors:
             if connector.metadata and connector.metadata.get("connectorBuildOptions", {}).get("baseImage"):
-                return connector
+                if not (connector.code_directory / "build_customization.py").exists():
+                    return connector
         pytest.skip("No connector with a connectorBuildOptions.baseImage metadata found")
 
+    @pytest.fixture
+    def connector_with_base_image_with_build_customization(self, connector_with_base_image):
```

Comment:
❓ seeing that our other fixture uses real connectors, in the future would we potentially change this to return all (real) connectors with build customizations?

Reply:
Yes why not! It will slow down the test but add coverage.
In any case I think we shall make an automation that will require a review from our team if a connector is implementing a build_customization module.